### PR TITLE
add IS-IS docs

### PR DESCRIPTION
**Pull request by jklare**
Signed-off-by: Jan Klare <jan.klare@bisdn.de>

### CODE REVIEW

File: network_configuration/isis.md [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+title: Intermediate System to Intermediate System (IS-IS)
+parent: Network Configuration
+---
+
+# Intermediate System to Intermediate System
+
+## Introduction
+
+IS-IS is an interior gateway protocol (IGP) widely used in large backbone
+provider networks and is defined in[ISO/IEC 10589:2002](https://www.iso.org/standard/30932.htm://www.iso.org/standard/30932.html).
```
Comment (hilmarm) [inline network_configuration/isis.md:11]:
```suggestion
provider networks and is defined in [ISO/IEC 10589:2002](https://www.iso.org/standard/30932.htm://www.iso.org/standard/30932.html).
```

File: network_configuration/isis.md [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+ |                                                             |       |                                                                 |
+ |                                                             |       |                                                                 |
+ +-------------------------------------------------------------+       +-----------------------------------------------------------------+
+```
+
+To allow all switches and servers to communicate with each other, we first have
+to make sure all directly connected elements share at least one common network
+for each connection. In the topology shown above, these networks will be
+`10.0.0.0/24`, `10.0.1.0/24` and `10.0.2.0/24`. The goal of this example is to
+exchange routes for the two networks `10.0.100.2/32` and `10.0.100.2/32`
```
Comment (hilmarm) [inline network_configuration/isis.md:65]:
```suggestion
exchange routes for the two networks `10.0.100.2/32` and `10.0.101.2/32`
```

File: network_configuration/isis.md [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+...
+isisd=yes
+...
+```
+
+The following configuration guide will explain how to configure a simple IS-IS
+toplogy containing two switches and two servers, each exchanging routes via
+IS-IS like shown below.
+
+```
```
Comment (hilmarm) [inline network_configuration/isis.md:35]: Possibly add port names to figure to match the configuration given below. I.e. port7 and port54

**Comment by hilmarm** [comment]
Looks good to me, except for the small comments I made above.

File: network_configuration/isis.md (hunk 3 above)
Comment (jklare) [reply network_configuration/isis.md:35]: done

**Review by rubensfig — APPROVED** [review]

**Pull request merged**